tests/chainerx_tests/unit_tests/routines_tests/test_manipulation.py

```diff
@@ -1,4 +1,5 @@
 import unittest
+from itertools import chain, combinations
```

Comment:
We do not allow importing non-module objects: https://docs.openstack.org/hacking/latest/user/hacking.html#imports
Please just import `itertools`.

Reply:
Done.